lib/Wikia/src/Service/User/Permissions/PermissionsAccessor.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Wikia\Service\User\Permissions;
+
+use Wikia\DependencyInjection\Injector;
+
+trait PermissionsAccessor {
+
+	/**
+	 * @var PermissionsService
+	 */
+	private $permissionsService;
+
+	/**
+	 * @return PermissionsService
+	 */
+	private function permissionsService() {
```

Comment:
I'd suggest `protected` here. Otherwise if class `A` uses `PermissionsAccessor` and the subclass `B` will also want to do it, it might have some problems IIRC.